api/FileSystemHandle.json

```diff
@@ -199,7 +199,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "111"
```

Comment:
I'm not sure whether this even exists in any spec... Anyway, not covered

Reply:
No, still a proposal https://github.com/whatwg/fs/pull/9